packages/datadog_flutter_plugin/lib/src/datadog_configuration.dart

```diff
@@ -262,6 +263,25 @@ class DdSdkConfiguration {
   /// value impacts the frequency of performing network requests by the SDK.
   UploadFrequency? uploadFrequency;
 
+  /// Sets the current version number of the application.
+  ///
+  /// By default, both iOS and Android sync their version numbers with the
+  /// current version in your pubspec, minus any build or pre-release
+  /// information. This property should only be used if you want to add this
+  /// additional information back in, or if the version in your pubspec does not
+  /// match your application version.
+  ///
+  /// Because 'version' is a Datadog tag, it needs to comply with the rules in
+  /// [Defining
+  /// Tags](https://docs.datadoghq.com/getting_started/tagging/#defining-tags)
+  /// Datadog documentation. We will automatically replace `+` with `-` for
```

Comment:
Should we say that if some other extra characters are presented in the version which don't comply with tag rules, they will be converted to underscores by the native SDKs?

Reply:
Does it do that for versions actually?

Reply:
yes, in case of logs version will be in the event in `ddtags` attribute which will be sanitized per rules.

For RUM it is different, because `version` is a part of query params there.

Reply:
I think since we already link the documentation there, we should be okay. I'm basically only documenting the non-default replacement case.